### DISCUSSION TITLE
https://greendolphindance.com/2024/07/25/%E4%B8%83%E6%9C%88%E7%A2%8E%E7%A2%8E%E5%BF%B5/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#28] [General]
# https://greendolphindance.com/2024/07/25/%E4%B8%83%E6%9C%88%E7%A2%8E%E7%A2%8E%E5%BF%B5/

Back to the rough ground!

https://greendolphindance.com/2024/07/25/%E4%B8%83%E6%9C%88%E7%A2%8E%E7%A2%8E%E5%BF%B5/

**Comment by redsherry** [comment]
写好长！羡慕输出的能力！

**Reply by greendolphindance** [reply]
啰里八嗦！

**Reply by redsherry** [reply]
爱看！请多写！

**Comment by wang7peng** [comment]
同性交友都挺 **纯粹** 的，有相同兴趣，陌生的时候无话不谈，熟了还互帮互助。异性社交就不太行，陌生的时候患得患失，熟了又开始想有的没的了。哈哈哈，口无遮拦的结果就是躺列~。之前翻 学霸大妹妹 的 [picx repo](https://github.com/greendolphindance/picx-images-hosting) 发现了 cv ，按上面号码搜到了 vx，本来也想加好友来着，现在想想，唐突的行为跟 ”妹妹陪吃饭“ 领导一样，肯定没有好果子吃，放弃了 😂
